app/components/tag_list_form_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<% if Spotlight::Engine.config.site_tags %>
+  <div class="form-group row">
+  <label class="col-form-label col-md-2" for="tag_list">Tag list</label>
+  <div class="col-md-10">
+    <div class="overflow-scroll rounded border px-3 py-2 h-25" style="overflow: scroll; height: 25vh!important;">
```

Comment:
This inline style can get removed as soon as we update to bootstrap 5. I can move this to a css file but it didn't seem to make sense due to the upgrade we are doing soon.